openspec/changes/add-vector-storage/tasks.md

```
@@ -0,0 +1,109 @@
+# Tasks: 向量存储与余弦相似度计算
+
+## 任务列表
+
+### 1. 数据模型升级 ✅
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)